lib/freeagent/bank_transaction_explanation.rb

```diff
@@ -4,11 +4,15 @@ class BankTransactionExplanation < Resource
 
     resource_methods :find, :filter
 
-    attr_accessor :bank_account, :description, :entry_type, :attachment, :paid_bill
+    attr_accessor :bank_account, :description, :entry_type, :attachment, :paid_bill, :paid_invoice, :bank_transaction
     decimal_accessor :gross_value
 
     date_accessor :dated_on
 
+    def self.find_all_by_bank_account(bank_account, options = {})
```

Comment:
Let's keep it simple for now, I don't remember seeing a lot of concerns in the gem codebase.